components/__tests__/platform-events.test.js

```diff
@@ -2,7 +2,7 @@ import { Events } from "../platform/platform-events";
 
 describe("Events", () => {
   test("should contain defined structure", () => {
-    expect(Events.length).toEqual(22);
+    expect(Events.length).toEqual(32);
```

Comment:
Hmm, I think that this test is not very good, because after adding new methods it crashes. Maybe we should test only if it's not empty?

Reply:
If we add or remove any events from this list it's important to align implementation for Android and iOS platform as both uses it - for me it looks like it's important to check at least amount of events on the list.

cc: @darek607